libs/components/forms/src/lib/modules/checkbox/checkbox.component.scss

```diff
@@ -5,6 +5,10 @@
   display: none;
 }
 
+.sky-checkbox-icon-indeterminate {
+  font-size: 14px;
```

Comment:
I carried this over from the tree view styles; however, did not carry over the quarter pixel adjustment as it did not have noticeable effect and the font size tweak helps with the centering issue on its own.